testsuite/functional-testsuite/src/test/script/prepare_siestajs.groovy

```diff
@@ -35,6 +35,22 @@ ant.unzip(src: artifacts['com.bryntum.siesta:siesta-standard'].file, dest: siest
   }
 }
 
+// patch phantomjs-launcher.js to allow screenshots
+def phantomjsLauncher = ant.fileScanner {
+  fileset(dir: siestaDir, includes: '**/phantomjs-launcher.js')
+}.collect{it}[0]
+
+def lines = phantomjsLauncher.readLines()
+lines.addAll(273,
```

Comment:
Can we search for the right place to insert this code, rather than rely on the magic `273` number?